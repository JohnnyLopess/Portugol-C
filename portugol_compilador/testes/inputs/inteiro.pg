programa
{
inicio
inteiro idade
idade = 20
escreva(idade)
fim
}