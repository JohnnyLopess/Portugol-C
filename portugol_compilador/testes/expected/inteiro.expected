#include <stdio.h>

int main() {
    int idade;
    idade = 20;
    printf("%d\n", idade);
    return 0;
}
